GitUI/HelperDialogs/FormResetCurrentBranch.cs

```diff
@@ -22,13 +22,10 @@ public enum ResetType
             Hard
         }
 
-        [Obsolete("For VS designer and translation test only. Do not remove.")]
-        private FormResetCurrentBranch()
-        {
-            InitializeComponent();
-        }
```

Comment:
Without a default ctor it is impossible to open the form in the designer

Reply:
Sorry, this was for those who can read. Restored.